common/rpc/interceptor/caller_info.go

```diff
@@ -35,32 +35,48 @@ import (
 )
 
 type (
-	CallerInfoInterceptor struct {
```

Comment:
I don't think you should rename this, it's an interceptor, and the name should reflect it IMHO.

Reply:
Go style guide suggests avoiding repeating the package name for exported symbols since the package name will always be visible. I don't think we use this struct inside this package, so it will always be written as `interceptor.CallerInfo`, which I think is still clear.